### ISSUE TITLE
See preferences without accessing through menu bar?

### ISSUE THREAD

**Issue by letitiayhho** [Clipy/Clipy#482]
Hi, like some others, I tried to hide clipy from my menu bar. Unfortunately, once you hide it from the menu bar you cannot access its preferences, not unless you delete some of its files, restart your computer, and reset Clipy. Can you make the preferences window open when you double click on the app in Applications, instead of requiring it to be accessed through the menu bar icon?

**Comment by a-lianos** [comment]
![image](https://user-images.githubusercontent.com/84623567/137307408-3d9b7761-219d-4bde-b68e-7f8f3e28bc41.png)

There is an option at the bottom of the menu.

**Comment by letitiayhho** [comment]
Mine doesn't have this option? I only see the history

**Comment by AmiciA** [comment]
I also have hidden the menu bar icon, and can only see the history itself in the context menu.

**Comment by letitiayhho** [comment]
Ah, I realize there are two separate shortcuts for accessing Clipy. One that shows only the history, and one that shows history and preferences. For my mac, the shortcut to show history and preferences is cmd + shift + v, to view only history use cmd + ctrl + v. I hope this helps!

**Comment by AmiciA** [comment]
I see that now—I had disabled all the shortcuts except for the history, because I only ever needed the history in the context menu. But if you do that, and then hide the menu bar icon, it seems like the only way to reopen the preferences window is to delete all Clipy data as shown in #459 and restart.

**Comment by letitiayhho** [comment]
Yeah, seems like it would be useful to be able to open the settings by clicking the app's icon, right now it doesn't do anything

**Comment by OleksiL** [comment]
This is very annoying. I also hide icons and disabled all other shortcuts except History - now I need to delete the application to change settings. It should show preferences on launch from applications. As Rectangle does, for example

**Comment by takeaki-m** [comment]
Hi everyone. I encountered the same issue and it was very annoying. I think I may have found one of the solutions at #101 . So please try this.  #101 is answered by Japanese, so I translate it with English

I resolved the problem by the solution 2, run the `defaults~` command


1. Using tools something like "AppCleaner" to uninstall Clipy, then re-install.
http://itea40.jp/application/mac-utilities/appcleaner/
2. If you can use command on Terminal, run the following command and restart Clipy
`defaults delete com.clipy-app.Clipy`
3. If your settings remain at default, run `Command + Shift + v` and then show menu.

**Comment by Econa77** [comment]
We are aware that there are cases where Clipy can no longer be opened after changing certain settings.

You can reset the settings with:

`defaults delete com.clipy-app.Clipy`

However, I also understand that deleting all existing settings is not ideal.

We are considering adding either an emergency shortcut that always opens Clipy, or a shortcut that directly opens the Preferences window. We will track this in #337, so if you are still interested in Clipy, please follow #337 for updates.

Thank you all for your help and feedback.
